Automattic/jetpack
# WordAds: Update loader snippets

Update js IPW loader snippets to support https://www.chromestatus.com/feature/6251880185331712

#### Changes proposed in this Pull Request:

* Update to the ad loader scripts

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Request here: p5UegX-xy-p2

#### Testing instructions:

1) Enable Jetpack Ads module on your test site (requires Premium sub).
2) View page with ads, check console has no errors (ads should continue loading as usual).

#### Proposed changelog entry for your changes:

* Update ad loader support for https://www.chromestatus.com/feature/6251880185331712
